examples/lock-app/nrfconnect/main/include/AppTask.h

```diff
@@ -67,7 +69,6 @@ class AppTask
     static void FunctionHandler(const AppEvent & event);
     static void StartBLEAdvertisementAndLockActionEventHandler(const AppEvent & event);
     static void LockActionEventHandler(const AppEvent & event);
-    static void StartBLEAdvertisementHandler(const AppEvent & event);
```

Comment:
Maybe it would be better to make `FabricTableDelegate` a friend of `AppTask` instead of moving this method to public.

Reply:
Oh, great point :)